### DISCUSSION TITLE
PR review changes comments lost when new commit

### DISCUSSION THREAD

**Discussion by TrueWill** [community/community#66148] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

If you're reviewing a PR (Review changes), start typing in the comment box, and the submitter adds a new commit, it appears to lose your comments. You get a message about reviewing new commits and lose everything you've typed.

The message is OK, but could you please preserve comments? Most of those are likely still relevant.

**Comment by marijnvanderhorst** [comment]
Just encountered the same issue after writing a relatively large comment. Why throw away the comment? Please let me just look at the new changes, adapt my intended comment where necessary, and re-submit afterward.

**Comment by abyrd** [comment]
This has the potential to destroy/lose large amounts of work. Fortunately I had only been working on the text for 10 minutes when this happened to me, but in other situations there could have been significantly more work lost. Please consider retaining the submitted text in this situation.
